src/pint/models/model_builder.py

```diff
@@ -330,6 +345,9 @@ def _pintify_parfile(self, parfile, allow_name_mixing=False):
             parfile_dict = parse_parfile(parfile)
         else:
             parfile_dict = parfile
+
+        parfile_dict = _replace_fdjump_in_parfile_dict(parfile_dict)
```

Comment:
Right now this seems to be a very special single case.  Is there a point to having a slightly more general dictionary (re)parsing step where other functions could be added?  Maybe that's too complex for this, but again a comment line would help.

Reply:
I think we shouldn't delay this for the general case. But I agree that implementing the general case will be useful.

I have added comments to this effect.